src/components/social-media-graphics/race/social-card.module.scss

```diff
@@ -0,0 +1,320 @@
+    margin-left: 45px;
+  }
+
+  .cases-header, .deaths-header {
+    justify-self: start;
+    align-self: end;
+    font-size: 13px; // ignore-style-rule
+  }
+
+  .bar-label {
+    justify-self: end;
+    align-self: center;
+    text-align: right;
+  }
+
+  .bar,
+  .bar-label {
+    margin-bottom: 8px;
+    font-size: 13px; // ignore-style-rule
+    line-height: 1.15;
+  }
+
+}
+
+.footer {
+  color: $color-slate-700;
+  font-size: 13px; // ignore-style-rule
+  line-height: 1.2;
+}
+
+.footer {
+  display: flex;
+  align-items: flex-end;
+  margin-left: 16px; // matches grid
+  justify-content: space-between;
+
+  .notes {
+    margin-top: 13px;
+    width: 730px;
+    &.national {
+      bottom: 0px;
+    }
+  }
+}
+
+.asterisk-footnote {
+  margin-top: 8px;
+}
+
+.square {
+  .footer {
+    .notes {
+      width: 525px;
+    }
+  }
+}
+
+.ctp-logo {
+  width: 26px;
+  margin-right: 5px;
+}
+
+.car-logo {
+  width: 43px;
+}
+
+.bar-container {
+  display: flex;
+  font-size: 13px; // ignore-style-rule
+  line-height: 1;
+  align-items: flex-start;
+
+  &>span {
+    margin-left: 5px;
+    margin-top: 6px;
+  }
+}
+
+.bar {
+  height: 25px;
+  width: 0px;
+  font-size: 13px; // ignore-style-rule
+  text-align: right;
+  margin-bottom: 10px;
+
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+
+  &.has-inner-label {
+    padding-right: 10px;
+  }
+
+  &.barBlack {
+    background: $qualitative-color-palette-300;
+  }
+  &.barLatinx {
+    background: $qualitative-color-palette-400;
+  }
+  &.barAll {
+    background: $color-slate-400;
+  }
+  &.barAsian {
+    background: $color-plum-800;
+    color: white;
+  }
+  &.barAian {
+    background: $qualitative-color-palette-600;
+    color: white;
+  }
+  &.barWhite {
+    background: $qualitative-color-palette-200;
+  }
+  &.barNhpi {
+    background: $qualitative-color-palette-500;
+    color: white;
+  }
+  &.barAPi {
+    background: $color-plum-800;
+    color: white;
+  }
+}
+
+.death-bar-spacer {
+  width: 45px;
+}
+
+.insufficient-data {
+  font-size: 13px;
+  line-height: 1;
+  padding-top: 6px;
+  color: $color-slate-600;
+
+  &.insufficient-data-deaths {
+    margin-left: 45px;
+  }
+}
+
+.square {
+  .bar {
+    height: 35px;
+  }
+  .bar-container {
+    span {
+      margin-left: 5px;
+      margin-top: 13px;
+    }
+  }
+  .death-bar-spacer {
+    width: 24px;
+  }
+  .insufficient-data.insufficient-data-deaths {
+    margin-left: 24px;
+    padding-top: 11px;
+  }
+}
+
+.no-data-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  @include type-size(300);
+  padding: 116px 180px; // ignore-style-rule
+
+  .logos-container {
+    display: flex;
+    margin-top: 16px;
+    justify-content: center;
+
+    .ctp-logo {
+      width: 26px;
+      margin-right: 6px;
+    }
+
+    .car-logo {
+      width: 43px;
+    }
+  }
+
+  &.square {
+    outline: 2px solid red;
+    border: 2px solid blue;
```

Comment:
todo: remove these
(This will need to happen after Sunday, when the QA checks are complete. Adding to the launch list.)